groups/committee-steering/groups.yaml

```
@@ -2,8 +2,9 @@ groups:
   - email-id: steering@knative.team
     name: steering
     description: |-
-      Private list for current and emeritus members of the Knative Steering Committee
+      Private list for current members of the Knative Steering Committee
```

Minor nit, but if anyone can post, should this just say "List for.."?
```suggestion
      List for current members of the Knative Steering Committee
```

Thanks for tracking this down @vaikas